hphp/test/quick/switch_case_missing_fallthrough_comment.php

```diff
@@ -0,0 +1,18 @@
+<?hh
```

Comment:
This test looks good, but it's in the wrong directory. `hphp/test/quick` is used for runtime tests that run quickly, whereas you want `hphp/hack/test/quickfixes`.